Src/IronPython/Runtime/Operations/BigIntegerOps.cs

```diff
@@ -715,8 +715,17 @@ private static bool IsInt32(this BigInteger self)
             return 1;
         }
 
-        public static int bit_length(BigInteger self) {
+        public static object bit_length(BigInteger self) {
+#if NET
+            long length = self.Sign switch {
```

Comment:
I guess this is needed since we don't have a .NET 5+ assembly for the DLR?

Reply:
Do you think it should be moved to the DLR?

Reply:
I'm assuming it produces exactly the same result as `MathUtils.BitLength` does, so from that point of view it would makes sense to have it there - but since this appears to be the only call site and we're not publishing a .NET 5+ DLR right now it's probably not worth it.